src/services/address-service.js

```diff
@@ -0,0 +1,89 @@
+// import axios from "axios";
+
+// const encodedToken = localStorage.getItem("token");
+
+const address = [
+    {
+        _id: Math.floor((Math.random() * 100) + 1),
```

Comment:
you can use uuid for getting random id's

Reply:
Ok